### ISSUE TITLE
JavaScript architecture. How to build and structure large-scale JavaScript apps

### ISSUE THREAD

**Issue by mikaelbr** [superherojs/superherojs#54]
We need resources for how to build and structure large-scale JavaScript applications, and general resources on good architecture.

**Comment by kimjoar** [comment]
Haven't had time to read this yet, but appears relevant: https://medium.com/@addyosmani/javascript-application-architecture-on-the-road-to-2015-d8125811101b

Anyone read it? Thoughts?

**Comment by mikaelbr** [comment]
I don't think it is superhero.js relevant. It's more a summary and prospectal than anything else. It's a good read though, but more time dependant. I saw a article that might be relevant earlier to day, but I didn't find it now. Shall look again when I'm on the computer.
